clang/utils/ci/buildkite-pipeline.yml

```diff
@@ -1,82 +0,0 @@
-#===----------------------------------------------------------------------===##
```

Comment:
`clang/utils/ci/run-buildbot` can be removed too, I think.

Reply:
I decided to keep it intact out of caution, and asked on Discord instead whether this file is still needed.

Reply:
I did check and it can be removed. I wrote that script so hopefully I’m not mistaken!

I will also go and clean up (remove the clang-ci pipeline on Buildkite, since it’s not used anymore). I will wait a week begore doing so to avoid breaking any ongoing PR that might trigger it.

Reply:
Committed as 681fc40d68936f145963207236d62886e5a34e5d